src/main/kotlin/io/snyk/plugin/ui/settings/ScanTypesPanel.kt

```diff
@@ -207,15 +207,20 @@ class ScanTypesPanel(
         }
     }
 
-    private fun checkSastEnable() {
+    fun checkSastEnabled(token: String? = pluginSettings().token) {
         setSnykCodeAvailability(false)
+        if (token.isNullOrBlank()) {
+            showSnykCodeAlert("A Snyk Token is necessary to check for Snyk Code enablement.")
```

Comment:
Where does this get shown? Screenshot would be great 🙏

Reply:
It's only shown if no token is set, as we can only check Snyk Code enablement with a working API token.